### PR TITLE
Enhancement: Add German translations

### PR DESCRIPTION
**Pull request by OskarStark**
cc @xabbuh

### CODE REVIEW

File: src/Resources/translations/VerifyEmailBundle.de.xlf [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" version="1.2">
+    <file original="file.ext" source-language="en" datatype="plaintext">
+        <body>
+            <trans-unit id="1">
+                <source>%count% year|%count% years</source>
+                <target>%count% Jahr|%count% Jahren</target>
```
Comment (xabbuh) [inline src/Resources/translations/VerifyEmailBundle.de.xlf:7]: In which context is this used? Depending on that `%count% Jahre` may be correct as well.

Comment (xabbuh) [reply src/Resources/translations/VerifyEmailBundle.de.xlf:7]: applies below as well

Comment (OskarStark) [reply src/Resources/translations/VerifyEmailBundle.de.xlf:7]: CleanShot 2021-01-20 at 11 42 12

In the confirmation email

**Review by xabbuh — APPROVED** [review]

**Review by jrushlow — APPROVED** [review]
Thank you @OskarStark for the new translations!

**Comment by weaverryan** [comment]
Thanks everyone!

**Pull request merged**